deploy/template.yaml

```diff
@@ -324,7 +344,7 @@ Resources:
                 Resource: !Join
                   - ""
                   - - !Sub "arn:aws:kms:${AWS::Region}:${AWS::AccountId}:key/"
-                    - !Sub "{{resolve:ssm:/${AWS::StackName}/KMS/JwtSigningKey/Id}}"
+                    - !Sub "{{resolve:ssm:/${FrontendStackName}/KMS/JwtSigningKey/Id}}"
```

Comment:
instead of supply FrontendStackName - could we do a split and join on the stackname to exclude the dev-1 part thus not requiring a parameter?

Reply:
As discussed, will keep as `FrontendStackName` parameter due to the complexity of splitting out `dev-1-` from `dev-1-account-management-frontend`.